components/kucoin_futures/actions/generic-api-call/generic-api-call.mjs

```diff
@@ -0,0 +1,51 @@
+import kucoin_futures from "../../kucoin_futures.app.mjs";
```

Comment:
please use cammelCase, this import should look like: 
`import kucoinFuturesApp from "../../kucoin_futures.app.mjs";`